### PR TITLE
feat(condo): DOMA-7578 added exclusions for bad words filter

### PR DESCRIPTION
**Pull request by Alllex202**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/news/constants/badWords.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { RU_LOCALE, EN_LOCALE } = require('@condo/domains/common/constants/locale')
+
+
+const badWordsExclusionsByLocale = {
```
Comment (AleX83Xpert) [inline apps/condo/domains/news/constants/badWords.js:4]: Why don't you create just a single array?
const BAD_WORDS_EXCLUSIONS = ['ипу']

**Review by AleX83Xpert — COMMENTED** [review]
Could you please add a test?
Are there a case-sensitive detected bad words?
`ипу` and `ИПУ` and `Ипу`

File: apps/condo/domains/news/constants/badWords.js (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/news/constants/badWords.js]: fixed

**Comment by Alllex202** [comment]
> Could you please add a test? Are there a case-sensitive detected bad words? `ипу` and `ИПУ` and `Ипу`

 - added test 
 - word checking is case insensitive

File: apps/condo/domains/news/utils/serverSchema/badWords.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * Check whether the input string contains bad words exclusions or not
+ *
+ * @param {string} word
+ * @return {boolean}
+ */
+const checkBadWordsExclusions = (word) => {
+    console.log({
+        BAD_WORDS_EXCLUSIONS_RE,
+    })
```
Comment (AleX83Xpert) [inline apps/condo/domains/news/utils/serverSchema/badWords.js:31]: Is this logging necessary?

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/news/utils/serverSchema/badWords.js (hunk 2 above)
Comment (Alllex202) [reply apps/condo/domains/news/utils/serverSchema/badWords.js]: Fixed :)

**Pull request merged**